tests/zenko_e2e/backbeat/s3SDK.js

```diff
@@ -0,0 +1,36 @@
+const S3 = require('aws-sdk').S3;
+
+const scalityS3Client = new S3({
+    accessKeyId: process.env.ZENKO_BACKBEAT_ACCESS_KEY,
+    secretAccessKey: process.env.ZENKO_BACKBEAT_SECRET_KEY,
+    sslEnabled: false,
+    endpoint: 'http://127.0.0.1:8000',
```

Comment:
Do we assume that, when the tests are ran, it's expected to have 8000 port-forwarded to localhost? Why not use `zenko.local` ?

Reply:
Either way is okay really but might be good to have a note of it in the doc.